### PR TITLE
Improve startup error when keys_dir is not available

### LINKED ISSUE #4346: Unclear error message when peer-key directory is not writeable
... at least mention the path that is the problem instead of:
```
15:40:56.518 [error] Error in process <0.1777.0> on node aeternity@localhost with exit value:
{{error,{badmatch,{error,erofs}},[{aec_keys,ensure_dir,1,[{file,"/app/apps/aecore/src/aec_keys.erl"},{line,515}]},{aec_keys,worker_init,3,[{file,"/app/apps/aecore/src/aec_keys.erl"},{line,360}]},{aec_keys,enter_worker,3,[{file,"/app/apps/aecore/src/aec_keys.erl"},{line,348}]}]},[{aec_keys,enter_worker,3,[{file,"/app/apps/aecore/src/aec_keys.erl"},{line,355}]}]}
...
```

### PR DESCRIPTION
**Pull request by hanssv**
Fixes #4346 

This PR is supported by the Æternity Foundation

### CODE REVIEW

**Review by dincho — APPROVED** [review]
thanks

File: apps/aecore/src/aec_keys.erl [hunk 1]
```diff
@@ -511,12 +512,19 @@ p_gen_new_peer(Pwd, PubFile, PrivFile) ->
...
-            ok = filelib:ensure_dir(KeysDirFile);
+            case ensure_dir_(KeysDir) of
+                ok  -> ok;
+                Err -> error({could_not_ensure_key_dir, KeysDir, Err})
+            end;
         true ->
             ok
     end.
 
+%% Note: `filelib:ensure_path/1` was introduced in OTP-25
```
Comment (dincho) [inline apps/aecore/src/aec_keys.erl:523]: how is this comment related 🤔

Comment (hanssv) [reply apps/aecore/src/aec_keys.erl:523]: It is just an explanation for the silly decision to use `ensure_dir` when `ensure_path` is what we want...

**Review by mitchelli — APPROVED** [review]
